manifests/repo/debian.pp

```diff
@@ -16,7 +16,7 @@
   apt::source { 'jenkins':
     location => $location,
     release  => 'binary/',
-    repos    => '',
+    repos    => 'main',
```

Comment:
Is this now still compatible with older puppetlabs/apt releases, or should we raise the minimal version in metadata.json to 10x?

Reply:
I don't see why it wouldn't be compatible.